rogue/src/main.rs

```diff
@@ -300,11 +301,16 @@ impl State {
         if DEBUG_HIGHLIGHT_FLOOR {
             self.debug_highlight_floor(ctx);
         }
+        if DEBUG_HIGHLIGHT_BLOOD {
+            self.debug_highlight_blood(ctx);
+        }
     }
 
-    fn run_map_indexing_system(&mut self) {
-        let mut mis = MapIndexingSystem{};
-        mis.run_now(&self.ecs);
+    fn run_synchronization_systems(&mut self) {
+        let mut mss = MapSynchronizationSystem{};
+        mss.run_now(&self.ecs);
+        let mut css = ColorSynchronizationSystem{};
```

Comment:
CSS!? One step closer to the JavaScript frontend :trollface:

Reply:
Omg, cursed.